Partial Custom Tabs update (#4082)
GoogleChrome/developer.chrome.com: site/en/docs/android/custom-tabs/integration-guide/index.md

You can also choose to make the Partial Custom Tab non-resizable. In this case, users will not have the ability to resize the Partial Custom Tab. If a value below 50% of the screen height is provided in the intent, Chrome will automatically adjust the Custom Tab to 50% of the screen height. Partial Custom Tabs are not supported when users are in multi-window or landscape mode.

The snippet below changes the launch height of the Custom Tab. CustomTabsIntent.Builder has a new method `setInitialActivityHeightPx(int initialHeightPx)` that allows you to define the initial launch height in pixels. As a prerequisite to using Partial Custom Tabs, you will need to [connect to the Custom Tabs Service](/docs/android/custom-tabs/integration-guide/#connect-to-the-custom-tabs-service) or launch the custom tabs intent using [Activity#startActivityForResult()](/reference/android/app/Activity#startActivityForResult(android.content.Intent,%20int)).

There are API's that do not work when building an Intent for Partial Custom Tabs:

-   CustomTabColorScheme.navigationBarColor
-   CustomTabColorScheme.navigationBarDividerColor

> Partial Custom Tabs will inherit the host app's color scheme for the UI components above. You will be responsible for ensuring visual consistency for these components before launching a Partial Custom Tab.

Reviewer: >UI components above

Which UI components are you talking about? The last thing I saw was a couple of properties. Please clarify this.

> These new APIs are available from Chrome M107 and onwards.

Reviewer: ```suggestion
These new APIs are available from Chrome 107 and onwards.
```

public static final int TAB_HIDDEN = 6;
```

```java
/**

> * Called when Custom Tab is resized in its height. This is applicable when users resize a 

Reviewer: ```suggestion
* Called when Custom Tab's height is resized. This applies when users resize a 
```
